toolbox/oidc_verifier.go

```diff
@@ -26,6 +27,7 @@ type verifierCache struct {
 	errorTolerance time.Duration
 	tokenURL       *url.URL
 	verifiers      map[string]cachedVerifier
+	verifiersMutex sync.RWMutex
```

Comment:
As we perform a fix of concurrency, I think it would be good to have a specific commit for this change.